tasks/plugins.yml

```diff
@@ -45,14 +45,20 @@
     path: "{{ jenkins_home }}/updates/default.json"
     regexp: "1d;$d"
 
+# Added ability to specify version of a plugin.
```

Comment:
No need for extra inline comments here; the PR and default has enough documentation for the change if someone wants the history.

Note that the example variable in defaults and in the README should be updated with these kinds of examples...

Reply:
✔️